### PR TITLE
Pulling helm chart from master branch

### PR DESCRIPTION
**Pull request by ConnorPMcCarthy**
Gitlab-ci is pulling helm chart from a branch that doesnt exist anymore. Change branch to "master"

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -88,7 +88,7 @@ review-app:
       ]
       EOF
       )
-    - git clone -b cmccarthy/pivcac --single-branch https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.login.gov/lg-public/identity-idp-helm-chart.git
+    - git clone -b master --single-branch https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.login.gov/lg-public/identity-idp-helm-chart.git
```
Comment (zachmargolis) [inline .gitlab-ci.yml:91]: we've migrated all our repos to use `main` as the default branch, is that a possibiltiy for this Gitlab project too?

Comment (ConnorPMcCarthy) [reply .gitlab-ci.yml:91]: We can make that change.

Comment (ConnorPMcCarthy) [reply .gitlab-ci.yml]: @zachmargolis Made the requested change. Default branch for the helm chart repo is now main.

**Review by zachmargolis — APPROVED** [review]
LGTM!

**Pull request merged**